cli/ballerina-tool/src/main/java/org/ballerinalang/tool/util/ToolUtil.java

```diff
@@ -426,8 +426,8 @@ public static void checkForUpdate(PrintStream printStream, String[] args) {
                     String latestVersion = currentVersion.getLatest(versions.stream().toArray(String[]::new));
                     if (!latestVersion.equals(version)) {
                         printStream.println();
-                        printStream.println("New Ballerina " + latestVersion + " version is available");
-                        printStream.println("Please use \"ballerina dist update\" command to update");
+                        printStream.println("A New Ballerina " + latestVersion + " version (1.0.0) is available.");
```

Comment:
```suggestion
                        printStream.println("A new version (" + latestVersion + ") of Ballerina is available.");
```

Reply:
Changed as follows:

"printStream.println("A new Ballerina version is available:" + latestVersion);"